cmd/lachesis/commands/run.go

```diff
@@ -122,8 +133,14 @@ func AddRunFlags(cmd *cobra.Command) {
 	cmd.Flags().String("datadir", config.Lachesis.DataDir, "Top-level directory for configuration and data")
 	cmd.Flags().String("log", config.Lachesis.LogLevel, "debug, info, warn, error, fatal, panic")
 	cmd.Flags().Bool("log2file", config.Log2file, "duplicate log output into file lachesis_<BindAddr>.log")
-	if runtime.GOOS != "windows" {
+	switch runtime.GOOS {
+	default:
+		cmd.Flags().Bool("syslog", config.Syslog, "duplicate log output into syslog")
+		fallthrough
+	case "plan9", "nacl":
 		cmd.Flags().String("pidfile", config.Pidfile, "pidfile location; /tmp/go-lachesis.pid by default")
+		fallthrough
```

Comment:
@Maxime2 these `fallthrough`s should be removed, and the logic around syslog and pidfile should exclude Windows and handle the plan9 & nacl conditions accordingly.